api/src/pcapi/templates/invoices/debit_note.html

```diff
@@ -0,0 +1,299 @@
+    </clipPath>
+  </defs>
+</svg>
+<h1>Note de débit</h1>
+<div class="customerInfo">
+  <p><b>Structure :</b> {{ reimbursement_point.managingOfferer.name }} - {{ reimbursement_point.managingOfferer.siren }}
+  </p>
+  <p><b>Point de remboursement :</b> {{ reimbursement_point_name }} - {{ reimbursement_point_iban }}</p>
+</div>
+<h3 class="coloredTitle">
+  Note de débit suite à la création d’incident de paiement.
+</h3>
+<h3>
+  Détail des offres remboursées{% if including_finance_incident %} et des incidents de paiement{% endif %} incluant la
+  contribution offreur,
+  appliquée selon le barème défini dans nos conditions générales de vente
+</h3>
+<table>
+  <thead>
+  <tr class="headColumn">
+    <th>Typologie</th>
+    <th>Taux de remboursement (%)</th>
+    <th>Montant des réservations validées (TTC)</th>
+    <th>Taux de contribution offreur (%)</th>
+    <th>Montant de la contribution offreur (TTC)</th>
+    <th>Montant remboursé (TTC)</th>
+  </tr>
+  </thead>
+  <tbody>
+  {% for group in groups %}
+    <tr class="coloredSection">
+      <td colspan="6" style="text-align:left"><b> {{ group.label }} </b></td>
+    </tr>
+    {% for line in group.lines %}
+      <tr>
+        <td class="headRow">{{ line.label }}</td>
+        <td>{{ line.rate|fr_percentage }}</td>
+        <td>{{ line.bookings_amount|fr_currency }} €</td>
+        <td>{{ line.contribution_rate|fr_percentage }}</td>
+        <td>{{ line.contributionAmount|fr_currency }} €</td>
+        <td>{{ line.reimbursedAmount|fr_currency_opposite }} €</td>
+      </tr>
+    {% endfor %}
+    <tr style="color: dimgrey">
+      <td class="headRow">SOUS-TOTAL</td>
+      <td></td>
+      <td>{{ group.used_bookings_subtotal|fr_currency }} €</td>
+      <td></td>
+      <td>{{ group.contribution_subtotal|fr_currency }} €</td>
+      <td>{{ group.reimbursed_amount_subtotal|fr_currency_opposite }} €</td>
+    </tr>
+  {% endfor %}
+  <tr class="bottomTableText">
+    <td class="headRow">TOTAL</td>
+    <td></td>
+    <td>{{ total_used_bookings_amount|fr_currency }} €</td>
+    <td></td>
+    <td>{{ total_contribution_amount|fr_currency }} €</td>
+    <td>{{ total_reimbursed_amount|fr_currency_opposite }} €</td>
+  </tr>
+  </tbody>
+</table>
+<div>
+  <p style="font-weight: bold">Modalités de règlement :</p>
+  <p>Cette note de débit est payable au pass Culture par virement bancaire aux coordonnées bancaires suivantes :</p>
+  <p><span style="font-weight: bold">IBAN :</span> FR6740031000010000479895Z93</p>
+  <p><span style="font-weight: bold">BIC :</span> CDCGFRPPXXX</p>
+  <p>Merci de rappeler le numéro de pièce lors de votre règlement.</p>
+  <p style="font-style: italic">Aucun escompte ne sera accordé en cas de paiement anticipé.</p>
+  <p style="font-style: italic">Pénalité de retard : 3 fois le taux d’intérêt légal par mois de retard (loi 92.1442 du
+    31/12/92)Tout incident de paiement est passible de pénalités de retard ainsi que de l’indemnité forfaitaire pour
```

Comment:
Manque un espace, et probablement un point : 

```suggestion
    31/12/92). Tout incident de paiement est passible de pénalités de retard ainsi que de l’indemnité forfaitaire pour
```